WPMedia/arc-themes-components
# Themes 1074: Metadata Component

## Ticket

- [THEMES-1074](https://arcpublishing.atlassian.net/browse/THEMES-1074)

## Description

Convert MetaData from engine-theme-sdk to arc-themes-components

## Acceptance Criteria

_copy from ticket_

## Test Steps

1. Checkout branch - git checkout THEMES-1074
2. Update dependencies - npm i
3. In your blocks repo, checkout branch - git checkout THEMES-1074
4. In your feature pack, run npx fusion start -f -l @wpmedia/default-output-block
5. Inspect the Metadata in the <head> of the page and verify that the information in the tags looks correct
6. Add an "og:image" url in the meta fields of the article-right-rail template, and verify that a resized version of the url appears in the metadata
7. Add a "twitter:image" url in the meta fields of the article-right-rail template, and verify that a resized version of the url appears in the metadata

Possible Test Pages:

- Article: http://localhost/pf/2022/08/11/on-oregons-central-coast-a-low-key-exploration-of-wineries-restaurants-and-nature/?_website=the-gazette 
- Video: http://localhost/pf/video/2023/03/16/rain-on-the-road/?_website=the-gazette 
- Gallery: http://localhost/pf/gallery/2022/12/05/trains-planes-and-automobiles/?_website=the-gazette 
- Author: http://localhost/pf/author/denise-rodriguez/?_website=the-gazette 
- Search: http://localhost/pf/search/global/?_website=the-gazette 
- Tag: http://localhost/pf/tags/travel/?_website=the-gazette 
- Section: http://localhost/pf/news/?_website=the-gazette 
- Homepage: http://localhost/pf/homepage/?_website=the-gazette 


## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1074]: https://arcpublishing.atlassian.net/browse/THEMES-1074?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
